docs/users/navigation/joysticks.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Joystick Navigation
+layout: default
+
+grand_parent: Users
+parent: Navigation
+nav_order: 2
+---
+
+# Joystick Navigation
+In addition to the normal navigation in OpenSpace using the keyboard and mouse it is also possible to navigate using a game controller, such as an Xbox controller, PS4 controller or a SpaceMouse. If you want to use any of these supported controllers you need to include their asset file into a profile. This can be done in the profile editor in the launcher by editing the profile to include one of the joystick assets. All of the joystick assets are located in the sub-folder <code>data/assets/util/joysticks</code> inside the OpenSpace folder. It is important that the right asset file is included for the type of game controller you are using. For example the wireless Xbox asset will not work with a non-wireless Xbox controller and vice versa.
```

Comment:
In addition to the normal navigation in OpenSpace, using the keyboard and mouse, it is also possible to navigate using a game controller, such as an Xbox controller, PS4 controller or a SpaceMouse. If you want to use any of these supported controllers you need to include their corresponding asset file in a profile. This can be done in the profile editor in the launcher by editing the profile to include one of the joystick assets. All of the joystick assets are located in the sub-folder <code>data/assets/util/joysticks</code> inside the OpenSpace folder. It is important to include the right asset file for the type of game controller you are using. For example, the wireless Xbox asset will not work with a non-wireless Xbox controller and vice versa.